qt/widgets/common/src/ManageUserDirectories.cpp

```diff
@@ -82,15 +100,21 @@ void ManageUserDirectories::loadProperties() {
   m_uiForm.leDefaultSave->setText(saveDir);
 }
 void ManageUserDirectories::saveProperties() {
-  QString newSearchArchive;
-  QString newDataDirs;
-  QString newUserDirs;
-  QString newSaveDir;
-
-  if (m_uiForm.ckSearchArchive->isChecked())
-    newSearchArchive = "On";
-  else
-    newSearchArchive = "Off";
+  QString newSearchArchive = m_uiForm.cbSearchArchive->currentText().toLower();
+  if (newSearchArchive == "all") {
+    // do nothing
+  } else if (newSearchArchive == "none") {
```

Comment:
Wouldn't it be easier if the options matched between the ManageUserDirectories UI and the Mantid properties file?